haskell.nix

```diff
@@ -13,41 +13,30 @@ in self-hs: super-hs:
       pkgs.haskell.lib.dontCheck
         (self-hs.callCabal2nix x y {});
 
+    # FIXME: hevm is broken in our current nixpkgs pin, so we pull a newer one
```

Comment:
Is this still required?

Reply:
I'm not sure, I think this is required in order to build the hevm wrapper (`eth-utils`) you added? Haven't tried building it with other nixpkgs versions tbh --  I took this from da0274f37.

Reply:
if we're using a recentish version of nixpkgs then I think we should be fine to remove this